loader/0002-quilt.mod.json.md

```diff
@@ -0,0 +1,419 @@
+
+## The `access_widener` field
+| Type         | Required |
+|--------------|----------|
+| Array/String | False    |
+
+A single or array of paths to accesswidener files relative to the root of the mod JAR.
+
+## The `minecraft` field
+| Type   | Required |
+|--------|----------|
+| Object | False    |
+
+Contains flags and options related to Minecraft specifically.
+
+### The `environment` field
+| Type   | Required |
+|--------|----------|
+| String | False    |
+
+Defines the environment(s) that this mod should be loaded on. Valid values are:
+* `"*"` — All environments (default)
+* `"client"` — The physical client
+* `"server"` — The dedicated server
+
+## Custom Elements
+In addition to the defined elements above, mods and libraries will be able to add their own elements to the quilt mod file. Mods will be expected to define up to one top-level element corresponding to their mod id. The element can be of any type, so that mods can define either a single value, array of values, or a sub-object.
+
+## Example
+An example quilt.mod.json:
+```json
+{
+    "schema_version": 1,
+    "quilt_loader": {
+        "group": "org.quiltmc",
+        "id": "example_mod",
+        "version": "1.0.0",
+        "entrypoints": {
+            "main": [
+                "org.quiltmc.example_mod.impl.ExampleMod",
+                "org.quiltmc.example_mod.impl.ExampleModNetworking"
+            ],
+            // Since we only have a single client endpoint, no array is needed.
+            "client": "org.quiltmc.example_mod.impl.client.ExampleModClient",
+        },
+        "depends": [
+            "quilt_networking_api",
+            "quilt_rendering_api",
+            {
+                "id": "modmenu",
+                "environment": "client"
+            }
+        ],
+        "breaks": [
+            {
+                "id": "sodium",
+                "versions": "*",
+                "reason": "Sodium does not implement the Quilt Rendering API."
+            },
+            {
+                "id": "some_random_library",
+                "versions": [
+                    "1.23.456", // A reason is not required
+                    {
+                        "versions": "<1.0.0",
+                        "reason": "Stable API required"
+                    },
+                    {
+                        "versions": "1.5.3",
+                        "reason": "Contains a game-breaking bug"
+                    }
+                ]
+            }
+        ],
+        "metadata": {
+            "name": "Quilt Example Mod",
+            "description": "An example mod for the Quilt ecosystem.",
+            "contributors": {
+                "Haven King": "Developer"
+            },
+            "contact": {
+                "homepage": "https://quiltmc.org/"
+            },
+            "license": "CC0-1.0",
+            "icon": "assets/modid/icon.png"
+        }
+    },
+    "mixins": [
```

Comment:
The spec names that field as `mixin` while here it's `mixins`, is it supposed to be `mixin` or `mixins`?